pycode/epidemiology/epidata/getDIVIData.py

```diff
@@ -243,7 +246,61 @@ def download_data_for_one_day(last_number, download_date):
                         date(2020, 12, 19): 5326,
                         date(2020, 12, 22): 5333,
                         date(2020, 12, 30): 5350,
+                        date(2021, 1, 5): 5364,
+                        date(2021, 1, 6): 5367,
+                        date(2021, 1, 9): 5373,
+                        date(2021, 1, 16): 5387,
+                        date(2021, 1, 19): 5393,
+                        date(2021, 1, 20): 5396,
+                        date(2021, 1, 23): 5402,
+                        date(2021, 2, 2): 5422,
+                        date(2021, 2, 8): 5435,
+                        date(2021, 2, 13): 5446,
+                        date(2021, 2, 19): 5460,
+                        date(2021, 2, 23): 5468,
+                        date(2021, 2, 24): 5471,
+                        date(2021, 2, 25): 5478,
+                        date(2021, 2, 26): 5481,
+                        date(2021, 3, 1): 5487,
+                        date(2021, 3, 4): 5507,
+                        date(2021, 3, 8): 5517,
+                        date(2021, 3, 10): 5524,
+                        date(2021, 3, 16): 5536,
+                        date(2021, 3, 18): 5541,
+                        date(2021, 3, 19): 5544,
+                        date(2021, 3, 26): 5558,
+                        date(2021, 3, 28): 5563,
+                        date(2021, 3, 30): 5568,
+                        date(2021, 3, 31): 5571,
+                        date(2021, 4, 1): 5574,
+                        date(2021, 4, 5): 5583,
+                        date(2021, 4, 9): 5595,
+                        date(2021, 4, 26): 5629,
+                        date(2021, 5, 6): 5649,
+                        date(2021, 5, 7): 5652,
+                        date(2021, 5, 8): 5655,
+                        date(2021, 5, 14): 5667,
+                        date(2021, 5, 18): 5677,
+                        date(2021, 5, 25): 5692,
+                        date(2021, 5, 27): 5698,
+                        date(2021, 5, 30): 5705,
+                        date(2021, 6, 6): 5719,
+                        date(2021, 6, 8): 5748,
+                        date(2021, 6, 19): 5773,
+                        date(2021, 6, 21): 5777,
+                        date(2021, 6, 25): 5786,
+                        date(2021, 6, 26): 5789,
+                        date(2021, 7, 17): 5831,
+                        date(2021, 7, 24): 5845,
+                        date(2021, 8, 7): 5873,
+                        date(2021, 8, 10): 5879,
+                        date(2021, 8, 12): 5883,
+                        date(2021, 8, 14): 5888,
+                        date(2021, 8, 15): 5891,
+                        date(2021, 8, 18): 5897,
+                        date(2021, 9, 1): 5925
```

Comment:
The program takes much longer if it has to search for the correct number all the time.  So this lines are not needed but helpfull if you want it fast(er). Furthermore, after a run the program tells you if it found new numbers and prints what you have to copy for the code. I agree this is not very nice. Any suggestions are welcome :-).

Reply:
If it is only a performance optimization, it's probably fine for now. I have no suggestions right now, since I haven't looked at what it does in detail.